README.md

```diff
@@ -43,7 +43,7 @@ The easiest way to run the tests is to launch a local server and open
 
 ## Sound credits
 
-["explosion12" by V-ktor](https://freesound.org/people/V-ktor/sounds/435413/)
-["Spacey 1up/Powerup" by GameAudio](https://freesound.org/people/GameAudio/sounds/220173/)
-["a shot" by Halgrimm](https://freesound.org/people/Halgrimm/sounds/156895/)
-["Captive Portal" by Me As](https://files.freemusicarchive.org/storage-freemusicarchive-org/music/no_curator/Captive_Portal/Toy_Sounds_Vol_1/Captive_Portal_-_02_-_Me_As.mp3)
+  - ["explosion12" by V-ktor](https://freesound.org/people/V-ktor/sounds/435413/)
+  - ["Spacey 1up/Powerup" by GameAudio](https://freesound.org/people/GameAudio/sounds/220173/)
+  - ["a shot" by Halgrimm](https://freesound.org/people/Halgrimm/sounds/156895/)
+  - ["Captive Portal" by Me As](https://files.freemusicarchive.org/storage-freemusicarchive-org/music/no_curator/Captive_Portal/Toy_Sounds_Vol_1/Captive_Portal_-_02_-_Me_As.mp3)
```

Comment:
we can make free sounds from this: https://jfxr.frozenfractal.com/
eventually we can get rid of the free sounds